containers/images/foreman/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos:7
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release && \
+    yum -y install http://yum.theforeman.org/releases/nightly/el7/x86_64/foreman-release.rpm && \
+    yum -y install https://fedorapeople.org/groups/katello/releases/yum/nightly/katello/el7/x86_64/katello-repos-latest.rpm && \
+    yum -y install foreman-release-scl && \
+    yum -y install qpid-tools && \
+    yum -y install foreman tfm-rubygem-puma && \
+    yum -y install tfm-rubygem-katello tfm-rubygem-foreman_ansible tfm-rubygem-foreman_discovery tfm-rubygem-foreman_hooks && \
+    yum -y install tfm-rubygem-foreman_memcache tfm-rubygem-foreman_openscap tfm-rubygem-foreman_remote_execution tfm-rubygem-foreman_templates && \
+    yum clean all
+
+COPY container-assets/settings.yaml /etc/foreman/settings.yaml
+COPY container-assets/katello.yaml /etc/foreman/plugins/katello.yaml
+COPY container-assets/foreman_memcache.yaml /etc/foreman/plugins/foreman_memcache.yaml
+
+# Generate apipie:cache:index
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install foreman-sqlite && \
+		yum clean all
+
+COPY container-assets/build/database.yml /etc/foreman/database.yml
+
+RUN foreman-rake db:create
+RUN foreman-rake db:migrate
+RUN foreman-rake apipie:cache:index
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y remove foreman-sqlite && \
+		yum clean all
```

Comment:
you should really clear the cache too by rm it

Reply:
When I deleted the cache it had no effect on image size in my local testing.

Reply:
this is what yum prints:
```
Maybe you want: rm -rf /var/cache/yum, to also free up space taken by orphaned data from disabled or removed repos
```